### PR TITLE
Set Up JWT Authorization in Django #73

### PR DESCRIPTION
**Pull request by SergeKry**

### CODE REVIEW

File: forum/forum/settings.py [hunk 1]
```diff
@@ -150,11 +152,33 @@
 
 REST_FRAMEWORK = {
     'DEFAULT_RENDERER_CLASSES': [
-        'rest_framework.renderers.JSONRenderer', 
+        'rest_framework.renderers.JSONRenderer',
+        'rest_framework_simplejwt.authentication.JWTAuthentication',
```
Comment (mehalyna) [inline forum/forum/settings.py:156]: You may explicitly define DEFAULT_AUTHENTICATION_CLASSES for clarity:
```
REST_FRAMEWORK = {
    'DEFAULT_RENDERER_CLASSES': [
        'rest_framework.renderers.JSONRenderer',
    ],
    'DEFAULT_AUTHENTICATION_CLASSES': [
        'rest_framework_simplejwt.authentication.JWTAuthentication',
    ],
}
```

File: forum/forum/settings.py [hunk 2]
```diff
@@ -150,11 +152,33 @@
...
 SECURE_SSL_REDIRECT = os.getenv('DJANGO_SECURE_SSL_REDIRECT', 'True') == 'True'
+
+
+# JWT settings
+
+SIMPLE_JWT = {
+    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=60),
+    'REFRESH_TOKEN_LIFETIME': timedelta(days=30),
+    'ROTATE_REFRESH_TOKENS': True,
+    'BLACKLIST_AFTER_ROTATION': False,
```
Comment (mehalyna) [inline forum/forum/settings.py:172]: Consider setting `BLACKLIST_AFTER_ROTATION` to `True` if you want to invalidate old refresh tokens after they are used.
If `BLACKLIST_AFTER_ROTATION` is `True`, ensure `rest_framework_simplejwt.token_blacklist` should be added to `INSTALLED_APPS`:
```
INSTALLED_APPS += [
    'rest_framework_simplejwt.token_blacklist',
]
```

File: forum/forum/settings.py [hunk 3]
```diff
@@ -150,11 +152,33 @@
...
+    'UPDATE_LAST_LOGIN': False,
+
+    'ALGORITHM': 'HS256',
+    'SIGNING_KEY': SECRET_KEY,
```
Comment (mehalyna) [inline forum/forum/settings.py:176]: Use a separate signing key for better security in production:
```
SIMPLE_JWT = {
    ...
    'SIGNING_KEY': os.getenv('JWT_SIGNING_KEY', SECRET_KEY),
    ...
}
```

**Review by mehalyna — CHANGES_REQUESTED** [review]

**Review by mehalyna — APPROVED** [review]

**Review by IraMladanovych02 — APPROVED** [review]

File: forum/forum/urls.py [hunk 4]
```diff
@@ -67,4 +67,6 @@ def api_test(request,format=None):
     path('',test,name='test'),
     path('swagger/', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
     path('redoc/', schema_view.with_ui('redoc', cache_timeout=0), name='schema-redoc'),
+    re_path(r'^auth/', include('djoser.urls')),
```
Comment (ne0ascorbinka) [inline forum/forum/urls.py:70]: Everything above seems legit, the only thing is idk if other links beside djoser.urls.jwt are necessary, but idk

**Review by ne0ascorbinka — APPROVED** [review]

**Pull request merged**